src/utilities/fetch.test.js

```diff
@@ -0,0 +1,45 @@
+import Fetch from 'utilities/fetch';
+import axiosMock from "axios";
+
+test('successful data fetching', async () => {
+    const itemData = {
+        "data": {
+            "data": [
+                {
+                    "id": 1
+                },
+                {
+                    "id": 2
+                },
+                {
+                    "id": 3
+                },
+            ]
+        }
+    };
+
+    axiosMock.get.mockResolvedValueOnce(itemData)
+    const url = '/api/items';
+    const fetchResult = await Fetch(url);
+
+    expect.assertions(3);
+    expect(axiosMock.get).toHaveBeenCalledTimes(1)
+    expect(axiosMock.get).toHaveBeenCalledWith(url)
+    expect(fetchResult).toEqual(itemData.data)
```

Comment:
This doesn't seem like an overly useful test since it's really just testing that the mock returns what you told it to return.